website/content/en/preview/getting-started/getting-started-with-terraform/_index.md

```diff
@@ -81,31 +89,45 @@ module "vpc" {
   one_nat_gateway_per_az = false
 
   private_subnet_tags = {
-    "kubernetes.io/cluster/${var.cluster_name}" = "owned"
-    "karpenter.sh/discovery" = var.cluster_name
+    "kubernetes.io/cluster/${local.cluster_name}" = "owned"
+    "karpenter.sh/discovery"                      = local.cluster_name # for Karpenter auto-discovery
   }
 }
 
 module "eks" {
```

Comment:
I believe we need another security group in the EKS module to allow the the Kube APIServer to talk to the Karpenter webhook.

```
  cluster_security_group_additional_rules = {
    ingress_nodes_karpenter_ports_tcp = {
      description                = "Karpenter readiness"
      protocol                   = "tcp"
      from_port                  = 8443
      to_port                    = 8443
      type                       = "ingress"
      source_node_security_group = true
    }
  }
```

Reply:
went a different route for now and just used the EKS cluster security group for simplicity